src/main/java/com/sap/oss/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -33,10 +34,10 @@ private OssFeatures() {
       = new ArtifactVersionsFeature("Released artifact versions");
 
   /**
-   * Holds a version string.
+   * Holds a released artifact version.
```

Comment:
Here you say it is a released artifact version but the feature name is "Artifact version". Do you actually mean that the feature holds a release artifact version? If so, it might be better to rename it to `RELEASED_ARTIFACT_VERSION` and update the name. If no, let's please drop "released" from javadoc.